src/post.proj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
+
+  <!-- provides a hook for executing targets after the product and tests have been built -->
+  <Target Name="Build" DependsOnTargets="$(PostBuildDependsOn)" />
```

Comment:
Fair enough it would only be an issue for folks that did a rebuild direction of post.msbuild or src\dir.proj. I do still think it is worth stubbing out those targets to support the common workflows.